Update README with logo and documentation (#61)
WayScience/coSMicQC: README.md

<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">

> # Single cell Morphology Quality Control

Reviewer: Would it make sense for the abbreviation / project name to also be mentioned here in a parenthetical?
```suggestion
# Single cell Morphology Quality Control (coSMicQC)
```

Author: Hmm, I am going to not add this, I think in the logo is includes this abbreviation so to me it feels a bit repetitive. We can change this later though!

[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)

> 

Reviewer: Suggestion to add Poetry and PyPI latest release.

```suggestion
[![Poetry](https://img.shields.io/endpoint?url=https://python-poetry.org/badge/v0.json)](https://python-poetry.org/)
![PyPI - Version](https://img.shields.io/pypi/v/cosmicqc)

```

Author: Awesome! I like those a lot! I have added them, thanks for the suggestion!

> 

Reviewer: Consider adding another line for separating the text from the badges to help increase legibility. I'm not sure what'd be best here, but felt that it might help! Feel free to leave things as is with the spacing if you don't feel the same.

Author: It looks like with the addition of poetry and pypi version that I have added another line at the bottom to separate the text, so I think I have implemented this. Thanks for the suggestion!

> :stars: Navigate the cosmos of single-cell morphology with confidence — coSMicQC keeps your data on course!

> coSMicQC is a software tool to evaluate converted single-cell morphology outputs from CytoTable.

Reviewer: Suggesting for specificity:
```suggestion
coSMicQC is a Python package to evaluate converted single-cell morphology outputs from CytoTable.
```

Author: Agreed, changed!

> When performing object detection in CellProfiler, it is very hard to get "perfect" compartment segmentation across a large high-throughput screen.

Reviewer: Consider moving to active tense here, avoid possible opinionated language, and pluralize screens (along with object detection software?).
```suggestion
It can be challenging to get "perfect" single-cell compartment segmentation across large high-throughput screens when performing object detection in CellProfiler (or similar software).
```

Author: Fair, I think my excitement might have biased the sentence there, thanks for catching! I will accept the change!

> Technical artifacts can occur during segmentation, including under or over segmentation or segmentation of background or smudges/bright artifacts.

Reviewer: Consider the following suggestion. Also, it feels like this sentence might flow better after "However..." below.
```suggestion
Technical artifacts can arise during segmentation, leading to issues such as under-segmentation, over-segmentation, or the erroneous segmentation of background noise, smudges, or bright artifacts.
```

Author: I think this sounds more clear and I like the use of erroneous! I will accept this change!

> In single-cell analysis, interesting phenotypes often emerge by looking at the morphological differences. However, the presence of technical outliers in the data can obscure these biological insights, undermining the validity of the findings.

Reviewer: ```suggestion
In single-cell analysis, intriguing phenotypes often emerge by examining morphological differences.
However, technical outliers in the data can obscure these biological insights, compromising the validity of the findings.
```

Author: I like how this is more concise, updating now!

> By utilizing specific CellProfiler extracted morphological features, which we show work best with AreaShape features, you can detect technically incorrect segmentations that you can either label or remove prior to further preprocessing steps (like with pycytominer).

Reviewer: ```suggestion
By utilizing specific morphological features extracted with CellProfiler, particularly AreaShape features, you can identify technically incorrect segmentations.
These can then be labeled or removed before further preprocessing steps, such as those performed with pycytominer.
```

Author: Much more concise, updating now!

## Installation

Install coSMicQC from [PyPI](https://pypi.org/project/coSMicQC/) or from source:

```shell
# install from pypi
pip install coSMicQC

# install directly from source
pip install git+https://github.com/WayScience/coSMicQC.git
```

## Contributing, Development, and Testing

> Please see [contributing.md](docs/source/contributing.md) for more details on contributions, development, and testing.

Reviewer: ```suggestion
Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more details on contributions, development, and testing.
```

Author: Thank you for catching that! Fixed!
